### PR TITLE
Beatmap Nomination Group: Resignations and changes

### PR DESCRIPTION
**Pull request by Naxesss**
Removals (resignations, inactivity):
Bara-
CelsiusLK
DE-CADE
Electoz
Evening
Kyubey
Len
neonat
Raymond
Squichu
Stjpa
ZZHBOY
Yuii-
Xexxar

osu!catch, osu!mania and osu!taiko additions:
Arrival (Taiko)
Benny- (Catch)
Jonawaga (Taiko)
Julie (Mania)
Kin (Taiko)
Kuo Kyoka (Mania)
Skylish (Taiko)
Stefan (Taiko)
Surono (Taiko)
thzz (Taiko)

Other:
Renamed Nao Tomori to Naotoshi
Renamed ChaosLitz to Chaoslitz
Moved -Kamikaze- above -Mo- in order
Moved German as additional language from Battle to Bonsai
Added osu!catch to F D Flourite

Note: All added users are hyperlinked to their respective profiles and have their additional languages applied.

### CODE REVIEW

File: wiki/People/Beatmap_Nomination_Group/en.md [hunk 1]
```diff
@@ -40,23 +40,20 @@ Also note: the game modes columns listed denotes which game mode(s) the user wou
...
 | [Bakari](https://osu.ppy.sh/u/597957)             | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | Russian                  |
-| [Bara-](https://osu.ppy.sh/u/2533040)             | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | Dutch                    |
-| [Battle](https://osu.ppy.sh/u/4037545)            | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | German                   |
-| [Bonsai](https://osu.ppy.sh/u/987334)             | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  |                          |
-| [CelsiusLK](https://osu.ppy.sh/u/1409012)         | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | German                   |
+| [Battle](https://osu.ppy.sh/u/4037545)            | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  |                          |
+| [Bonsai](https://osu.ppy.sh/u/987334)             | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | German                   |
+| [Benny-](https://osu.ppy.sh/u/4023183)            | ![No][Ns]  | ![No][Nt]  | ![Yes][Yf] | ![No][Nm]  | Norwegian                |
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nomination_Group/en.md]: move this one up one (`Be` comes before `Bo`)

File: wiki/People/Beatmap_Nomination_Group/en.md [hunk 2]
```diff
@@ -40,23 +40,20 @@ Also note: the game modes columns listed denotes which game mode(s) the user wou
...
 | [ChaosLitz](https://osu.ppy.sh/u/3621552)         | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | Cantonese, Chinese       |
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nomination_Group/en.md]: correct the name, `Chaoslitz`, without the capital `L`

File: wiki/People/Beatmap_Nomination_Group/en.md [hunk 3]
```diff
@@ -66,25 +63,26 @@ Also note: the game modes columns listed denotes which game mode(s) the user wou
...
 | [Kibbleru](https://osu.ppy.sh/u/3193504)          | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  |                          |
+| [Kin](https://osu.ppy.sh/u/480689)                | ![No][Ns]  | ![Yes][Yt] | ![No][Nf]  | ![No][Nm]  | French                   |
 | [Koiyuki](https://osu.ppy.sh/u/2433507)           | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | Chinese, some Japanese   |
+| [Kuo Kyoka](https://osu.ppy.sh/u/2596306)         | ![No][Ns]  | ![No][Nt]  | ![No][Nf]  | ![Yes][Ym] | Vietnamese               |
 | [Kurai](https://osu.ppy.sh/u/77089)               | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | French                   |
-| [Kyubey](https://osu.ppy.sh/u/2195646)            | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | Russian                  |
 | [Lasse](https://osu.ppy.sh/u/896613)              | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | German                   |
 | [Len](https://osu.ppy.sh/u/1686145)               | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | Korean                   |
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nomination_Group/en.md]: ❓ Len is no longer listed in <https://osu.ppy.sh/g/28>

File: wiki/People/Beatmap_Nomination_Group/en.md [hunk 4]
```diff
@@ -40,23 +40,20 @@ Also note: the game modes columns listed denotes which game mode(s) the user wou
...
-| [DE-CADE](https://osu.ppy.sh/u/3734394)           | ![No][Ns]  | ![No][Nt]  | ![No][Nf]  | ![Yes][Ym] | Indonesian               |
 | [Delis](https://osu.ppy.sh/u/1603923)             | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | Japanese                 |
 | [Dellvangel](https://osu.ppy.sh/u/5186244)        | ![No][Ns]  | ![No][Nt]  | ![No][Nf]  | ![Yes][Ym] | Indonesian               |
 | [Doormat](https://osu.ppy.sh/u/3230571)           | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  |                          |
-| [Electoz](https://osu.ppy.sh/u/6485263)           | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | Thai                     |
-| [Evening](https://osu.ppy.sh/u/2193881)           | ![No][Ns]  | ![No][Nt]  | ![No][Nf]  | ![Yes][Ym] |                          |
 | [ezek](https://osu.ppy.sh/u/180241)               | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | Spanish                  |
 | [F D Flourite](https://osu.ppy.sh/u/2459589)      | ![Yes][Ys] | ![No][Nt]  | ![No][Nf]  | ![No][Nm]  | Chinese                  |
```
Comment (Xinnoh) [inline wiki/People/Beatmap_Nomination_Group/en.md]: Flourite is also a Ctb BN

**Pull request merged**